larpandoracontent/LArControlFlow/BdtBeamParticleIdTool.h

```diff
@@ -0,0 +1,505 @@
+/**
+ *  @file   larpandoracontent/LArControlFlow/BdtBeamParticleIdTool.h
+ *
+ *  @brief  Header file for the beam particle id tool class.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_BDT_BEAM_PARTICLE_ID_TOOL_H
+#define LAR_BDT_BEAM_PARTICLE_ID_TOOL_H 1
+
+#include "larpandoracontent/LArControlFlow/MasterAlgorithm.h"
```

Comment:
Does this need to know about a pandora::Algorithm, or specifically the MasterAlgorithm?